logger/logger_test.go

```diff
@@ -0,0 +1,89 @@
+package logger_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	log "github.com/mainflux/mainflux/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInfo(t *testing.T) {
+	cases := map[string]struct {
+		input string
```

Comment:
The test is missing the expected output. I would recommend to restructure it in such a way that is easy to acquire the contents and verify it, i.e. provide an implementation of `io.Writer` for test purposes.